### ISSUE TITLE
Upgrading from 7.x to 8.x not finding svg files

### ISSUE THREAD

**Issue by kiwidude68** [kreuzerk/svg-to-ts#154]
Hi @kreuzerk 

This is what I have which works perfectly fine using svg-to-ts 7.1.1 - all package.json based:

```
  "scripts": {
   "build:icons": "svg-to-ts",
```

Then further down...

```
 "svg-to-ts": {
    "conversionType": "files",
    "srcFiles": [
      "./libs/shared/ui-icons/src/icons/**/*.svg"
    ],
    "outputDirectory": "./libs/shared/ui-icons/src/icons",
    "prefix": "ppIcon",
    "typeName": "PpIconName",
    "interfaceName": "PpIcon",
    "fileName": "pp-icons",
    "modelFileName": "pp-icon.model",
    "svgoConfig": {
      "plugins": [
        "removeDimensions"
      ]
    }
  },

```
Based on the release notes for 8.x I tried just changing those two "svg-to-ts" places to "svg-to-ts-files" and removed the "conversionType" line. Would you expect I have to do anything else?

Because when I now run it I get an error message about it not finding the .svg files - that message being what you added in 8.2.0 and prevents issues I saw with 8.0/8.1 of a syntax error which was reported as a separate issue.

So it appears there is some sort of path change introduced by 8.x to do with the srcFiles between versions? Any suggestions how to work around it?

**Comment by kreuzerk** [comment]
Hi @kiwidude68. Thx a lot for this issue. If I understood you correctly you changed the binary and the config key.   You only have to change the binary from svg-to-ts to svg-to-ts-files. The key of the config object should still be svg-to-ts.

**Comment by kiwidude68** [comment]
Hi @kreuzerk - aha that's my bad, thanks for the steer. Having done that it now works. What steered me wrong was I initially tried to jump from 6.0.0 to 8.3.0 where I *did* just change the binary and not the key of the config object. However I had a different error at that time of "plugin name not found" or something like that, which had me then trying to change the key name too. 

As it turned out (as I rolled my way back to 6.0.0 and upgraded through versions one by one) there was an incompatibility with the format of the svgoConfig plugins section - the way that json was previously set by us (code I inherited, and worked perfectly fine in 6.x) was the plugins section looking like this:
```

   "svgoConfig": {
      "plugins": [ 
          { "removeDimensions": true }
      ]
```
Once I changed that to what is in my original post above the plugin name error with 7.x went away - and then I forgot when I retried 8.x to try not renaming that svg-to-ts config key...

It's been a long day! Thanks again for your help, I am out the other side at last...
